ios/FluentUI/Badge Field/BadgeView.swift

```diff
@@ -385,12 +394,13 @@ open class BadgeView: UIView {
         backgroundView.frame = bounds
 
         if let customViewSize = customViewSize(for: frame.size), customViewSize != .zero {
-			let customViewOrigin = CGPoint(x: customViewPadding.left, y: (frame.height - customViewSize.height) / 2)
+            let customViewOrigin = CGPoint(x: customViewPadding.left, y: (frame.height - customViewSize.height) / 2)
             dataSource?.customView?.frame = CGRect(origin: customViewOrigin, size: customViewSize)
-			let labelOrigin = CGPoint(x: customViewPadding.left + customViewPadding.right + customViewSize.width, y: (frame.height - labelSize.height) / 2)
-            label.frame = CGRect(origin: labelOrigin, size: labelSize)
+            let labelOrigin = CGPoint(x: customViewPadding.left + customViewPadding.right + customViewSize.width, y: (frame.height - labelSize.height) / 2)
+            let labelSizeThatFits = CGSize(width: frame.size.width - labelOrigin.x, height: labelSize.height)
+            label.frame = CGRect(origin: labelOrigin, size: labelSizeThatFits)
         } else {
-            label.frame = bounds.insetBy(dx: -size.horizontalPadding, dy: -size.verticalPadding)
+            label.frame = bounds.insetBy(dx: size.horizontalPadding, dy: size.verticalPadding)
```

Comment:
it's required on big text mode, otherwise the layout will overlap over border

| before | after  |
| ------------- | ------------- |
| ![a-default-inset](https://user-images.githubusercontent.com/37874286/178309352-d66f4845-b9cf-4c41-81fb-d4dcc7c39bf7.png) | ![a-updated-inset](https://user-images.githubusercontent.com/37874286/178309543-42b98caf-1a50-427e-ac01-699b1633a64f.png) |
| ![default-inset](https://user-images.githubusercontent.com/37874286/178309635-0b7ef78a-5217-4ddb-99f5-15695143937d.png) | ![updated-inset](https://user-images.githubusercontent.com/37874286/178309686-fba8cf47-6a70-4532-b9d0-24e275076c91.png) |